### ISSUE TITLE
devtools-v6 chrome extension brings page style change

### ISSUE THREAD

**Issue by YeAhzjx** [vuejs/devtools-v6#2198]
my project is vue 2.6.14,so when i use latest devtool is will console error:'Vue DevTools v7 detected in your Vue2 project. v7 only supports Vue3 and will not work.
prepare.js:1 The legacy version that supports both Vue 2 and Vue 3 has been moved to  https://chromewebstore.google.com/detail/vuejs-devtools/iaajmlceplecbljialhhkmedjlpdblhp
prepare.js:1 Please install and enable only the legacy version for your Vue2 app.'
then i change to legacy version,devtool is work,but,my project style has been broken
bug picture：
![dev-tools-bug](https://github.com/user-attachments/assets/bc9ddd2a-272d-4aa6-96ac-545d1537c657)

**Comment by alexzhang1030** [comment]
Hey, any details?

**Comment by YeAhzjx** [comment]
> Hey, any details?

May I ask what detailed information you need? I will certainly try my best to provide it if I can.

**Comment by YeAhzjx** [comment]
when i use this version:
![PixPin_2024-12-23_09-42-38](https://github.com/user-attachments/assets/de488089-9c5c-4b34-9f13-acd0ea4b3daf)
i get these:
![PixPin_2024-12-23_09-43-28](https://github.com/user-attachments/assets/d71b087e-2090-4c87-bf7e-de25d8711e4d)
![PixPin_2024-12-23_09-43-45](https://github.com/user-attachments/assets/a942309a-e399-4489-902a-31d5316273d9)
![PixPin_2024-12-23_09-44-07](https://github.com/user-attachments/assets/09fc28b7-3e7f-4a2f-918e-3f01a9147309)

when i use this version:
![PixPin_2024-12-23_09-44-41](https://github.com/user-attachments/assets/312e694d-9412-46c0-ad31-c248af129b4e)
i get these:
![PixPin_2024-12-23_09-45-05](https://github.com/user-attachments/assets/caf9d1e6-1b11-4b83-9814-62b2b2cb8e2e)
![PixPin_2024-12-23_09-45-18](https://github.com/user-attachments/assets/d956dc65-228b-4bf5-84e2-b44c5f596084)
![PixPin_2024-12-23_09-45-30](https://github.com/user-attachments/assets/e7e478bb-043d-4ed7-b8e9-be4def50a779)


In other words, if I use version 6.64, my project can use devtool but the style will be broken, if I use version 7.6.8, my project style will be normal but devtool will not be used

**Comment by alexzhang1030** [comment]
It's probably a devtools-v6 issue, So I'm gonna transfer this issue to the target repo. Let's talk there.

**Comment by YeAhzjx** [comment]
> It's probably a devtools-v6 issue, So I'm gonna transfer this issue to the target repo. Let's talk there.

I went through the console, only to notice that it was missing the '<style>html{font-size:100px; }</style>'
